src/gui/CMakeLists.txt

```diff
@@ -188,33 +186,9 @@ add_library(qbt_gui STATIC
     torrentcreatordialog.ui
     trackerentriesdialog.ui
     updownratiodialog.ui
-
-    # resources
-    about.qrc
 )
 
-if (WIN32 OR APPLE)
-    target_sources(qbt_gui PRIVATE programupdater.h programupdater.cpp)
-endif()
-
-if (UNIX AND Qt5DBus_FOUND)
-    target_link_libraries(qbt_gui PRIVATE Qt5::DBus)
-
-    target_sources(qbt_gui PRIVATE
-        qtnotify/notifications.h
-        qtnotify/notifications.cpp
-    )
-
-    find_package(X11)
-    if (X11_FOUND)
-        target_sources(qbt_gui PRIVATE
-            powermanagement/powermanagement_x11.h
-            powermanagement/powermanagement_x11.cpp
-        )
-    endif()
-endif()
-
-target_include_directories(qbt_gui PRIVATE ${CMAKE_CURRENT_SOURCE_DIR})
+target_sources(qbt_gui INTERFACE about.qrc)
```

Comment:
Please explain the changes like this one.

Reply:
Similar reasoning to https://github.com/qbittorrent/qBittorrent/pull/13327#discussion_r485134005. `app` needs access to this. In this new architecture, if this is `PRIVATE`, you'll end up with missing/broken stuff in the `About` dialog.